src/effects/effectchainslot.cpp

```diff
@@ -44,6 +44,10 @@ EffectChainSlot::EffectChainSlot(EffectRack* pRack, const QString& group,
     m_pControlChainSuperParameter->set(0.0);
     m_pControlChainSuperParameter->setDefaultValue(0.0);
 
+    m_pControlChainSuperForceUpdate = new ControlObject(ConfigKey(m_group, "super_force_next_update"));
```

Comment:
> Dos it make sense to disable softtakover if the effect is disabled?

I have played with it and I feel this is the right approach. The point of soft takeover is to prevent sudden audible jumps. If the sudden jumps are not audible, they are not a problem. Rather, it is confusing if the user has to turn the metaknob to another value before the linked parameter starts changing. This should only be done to prevent odd jumps in the playing sound.